assets/templates/_ModeBarDash.hbs

```diff
@@ -15,6 +15,26 @@
                 <span class="rtl-switcher">RTL</span>
             </li>
 
+            <li>
+                <a href="#" class="more-menu-trigger">
```

Comment:
"More" looks good and opens menu but close functionality needs to be added.

Reply:
I skipped this, its not implemented in the other layouts either and don't want mess around with handlebars too much before we simplify... it works as expected in the platform-client though.